dspace-server-webapp/src/test/java/org/dspace/app/rest/WorkspaceItemRestRepositoryIT.java

```diff
@@ -4666,4 +4610,380 @@ public void findOneFullProjectionTest() throws Exception {
                                 .andExpect(jsonPath("$._embedded.collection._embedded.adminGroup").doesNotExist());
 
     }
+
+    @Test
+    public void uploadBitstreamWithoutAccessConditions() throws Exception {
+        context.turnOffAuthorisationSystem();
+        Community community = CommunityBuilder.createCommunity(context).withName("Com").build();
+        Collection collection = CollectionBuilder.createCollection(context, community).withName("Col").build();
+        WorkspaceItem wItem = WorkspaceItemBuilder.createWorkspaceItem(context, collection).build();
+        context.restoreAuthSystemState();
+
+        // prepare dummy bitstream
+        InputStream pdf = getClass().getResourceAsStream("simple-article.pdf");
+        final MockMultipartFile pdfFile = new MockMultipartFile("file", "/local/path/simple-article.pdf",
+            "application/pdf", pdf);
+
+        // auth
+        String authToken = getAuthToken(eperson.getEmail(), password);
+
+        // upload file and verify response
+        getClient(authToken)
+            .perform(fileUpload("/api/submission/workspaceitems/" + wItem.getID()).file(pdfFile))
+            .andExpect(status().isCreated())
+            .andExpect(jsonPath("$.sections.upload.files[0].accessConditions", empty()));
+
+        // verify that access conditions have been persisted
+        getClient(authToken)
+            .perform(get("/api/submission/workspaceitems/" + wItem.getID()))
+            .andExpect(status().isOk())
+            .andExpect(jsonPath("$.sections.upload.files[0].accessConditions", empty()));
+    }
+
+    // The REST endpoint doesn't seem to support uploading a bitstream with the access conditions
+    @Test
+    @Ignore
+    public void uploadBitstreamWithAccessConditionOpenAccess() throws Exception {
```

Comment:
I wonder if we should just remove this ignored test.  If I recall correctly, we had planned for this to be a two-step process. You upload the Bitstream, then restrict it. So, I think this scenario is already covered by the next method `patchBitstreamWithAccessConditionOpenAccess()`, which does just that...uploads the file, then restricts it.

Reply:
We had planned to add tests for verifying the permissions can be set during the bitstream upload, but noticed this was never supported during the file upload.
I assumed this was just forgotten during the implementation of the file upload, but if there's no intention to support it, the test will never be relevant
I've removed the test now